tests/appsec/rasp/test_cmdi.py

```diff
@@ -30,8 +30,14 @@ def test_cmdi_get(self):
             self.r,
             "rasp-932-110",
             {
-                "resource": {"address": "server.sys.exec.cmd", "value": "/bin/evilCommand"},
-                "params": {"address": "server.request.query", "value": "/bin/evilCommand"},
+                "resource": {
+                    "address": "server.sys.exec.cmd",
+                    "value": "/usr/bin/reboot -f",
```

Comment:
If a test fails blocking the command call, would the host machine reboot?

Reply:
Yes, while as @jandro996 suggested, the reboot command will not usually run without admin permissions, using /usr/bin/touch seems like a good and safer choice. Thanks!

Reply:
Done